lib/tasks/Travis.rake

```diff
@@ -0,0 +1,7 @@
+desc 'Single task to run on CI'
```

Comment:
Can you give this a clearer description? 'Single task to run on CI' is accurate, but it doesn't tell me if it's a task that exists purely as a smokecheck, if it does something, etc.

The `desc` on tasks is what you see when you run `bundle exec rake --tasks`, so it's an important one-liner so that future devs can see what all the tasks do without having to actually open up the task code themselves.

Reply:
☘️